pkg/manifests/manifests.go

```diff
@@ -133,7 +133,7 @@ func tunedRenderedProfiles(tuned *tunedv1.Tuned, m map[string]tunedv1.TunedProfi
 					if *v.Data == *existingProfile.Data {
 						klog.Infof("duplicate profiles names %s but they have the same contents", *v.Name)
 					} else {
-						klog.Warningf("WARNING: duplicate profiles named %s with different contents", *v.Name)
+						klog.Errorf("ERROR: duplicate profiles named %s with different contents!", *v.Name)
```

Comment:
Is here a way to print the Tuned object name the profile comes from too? And maybe the profile that was selected (+ its parent Tuned) at the end of the loop?

Reply:
> Is here a way to print the Tuned object name the profile comes from too? 
Yes. I added it to the PR.

> And maybe the profile that was selected (+ its parent Tuned) at the end of the loop?
Not sure what exactly do you mean, but I'm sensing it would require larger code changes and hopefully the above is sufficient.